src/games/strategy/triplea/ai/fastAI/FastOddsEstimator.java

```diff
@@ -16,14 +16,19 @@
 
 public class FastOddsEstimator implements IOddsCalculator {
 
-  private GameData data;
-  private PlayerID attacker = null;
-  private PlayerID defender = null;
+  @SuppressWarnings("unused")
+private GameData data;
```

Comment:
formatting, the formatter is in: https://github.com/triplea-game/triplea/tree/master/eclipse/format